### PR TITLE
Make airbrake optional & more

### PR DESCRIPTION
**Pull request by tomdemeyer**
Updated readme, support disabling GA, nl locale...
You should revert the change to yp_login.js, though. I don't know how to keep that commit from this pull request..

### CODE REVIEW

**Comment by tomdemeyer** [comment]
Ah, also disable b81602a 
I'm no git power user

File: client_app/src/yp-user/yp-login.html [hunk 1]
```diff
@@ -231,9 +231,9 @@
               <div class="btn-auth btn-facebook" on-tap="_facebookLogin" hidden$="[[!hasFacebookLogin]]">
                 [[t('user.facebookLogin')]]
               </div>
-              <div class="islandIs" on-tap="_islandIsLogin" hidden$="[[!hasSamlLogin]]">
+              <!-- <div class="islandIs" on-tap="_islandIsLogin" hidden$="[[!hasSamlLogin]]">
```
Comment (rbjarnason) [inline client_app/src/yp-user/yp-login.html]: You should not have to comments this out - just make sure in the domain API settings that the SAML key fields are empty.

File: client_app/index_bi.html [hunk 2]
```diff
@@ -93,11 +93,14 @@
       });
     }
 
-    // Setup Google Analytics
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+    if( ! process.env.DISABLE_GA) {
```
Comment (rbjarnason) [inline client_app/index_bi.html]: This is not going to work - as this is an index.html file that is executed on the client browsers and is not in the node environment. Same goes for the other html files.

**Comment by tomdemeyer** [comment]
Ah :)
I had commented it out prevously and am getting a cached version, apparently. 
Or worse, still serving from the wrong directory.. I'll check, thanks.

File: client_app/src/yp-user/yp-login.html (hunk 1 above)
Comment (tomdemeyer) [reply client_app/src/yp-user/yp-login.html]: ok. will check

**Comment by rbjarnason** [comment]
Looks good! :+1:

**Comment by rbjarnason** [comment]
You have closed it?

**Comment by tomdemeyer** [comment]
you’ll get a new one :)


--

Tom Demeyer
Waag Society
http://waag.org

**Pull request closed without merging**